apps/src/lib/levelbuilder/course-editor/ResourcesEditor.js

```diff
@@ -80,6 +80,15 @@ export default class ResourcesEditor extends Component {
       ({type, link}) => link && type
     );
 
+    // If using migrated resources, we have to have a course version id
+    if (useMigratedResources && !this.props.courseVersionId) {
+      return (
+        <strong>
+          Cannot add resources to migrated script without course version.
```

Comment:
Can you add a help tip that explains how to make sure your script has a course version?